<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="reciever_forward">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reciever_left">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reciever_back">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reciever_right">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>620</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="reciever">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Приемник</string>
   </property>
  </widget>
  <widget class="QLabel" name="source">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Источник</string>
   </property>
  </widget>
  <widget class="QPushButton" name="source_left">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="source_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="source_forward">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="source_right">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QLabel" name="object">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Объект</string>
   </property>
  </widget>
  <widget class="QPushButton" name="object_up">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="object_down">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="object_right_rotate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>right rotate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="object_left_rotate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>left rotate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="single_shoot">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>620</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Single Shoot</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="I">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>650</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>680</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>I,мкА</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>t,мс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>U,кВ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="U">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>620</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>82</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="webcam_on">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Включить Web- камеру</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>637</width>
      <height>367</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="traject_source_left">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_source_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_source_forward">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_source_right">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_object_up">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_object_down">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_receiver_forward">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_receiver_back">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_receiver_left">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traject_receiver_right">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_source_left">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_object_up">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_object_down">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_receiver_right">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_source_right">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_source_back">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_source_forward">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_receiver_forward">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_receiver_back">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_for_receiver_left">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>reciever_forward</zorder>
  <zorder>reciever_left</zorder>
  <zorder>reciever_back</zorder>
  <zorder>reciever_right</zorder>
  <zorder>pushButton</zorder>
  <zorder>source</zorder>
  <zorder>source_left</zorder>
  <zorder>source_back</zorder>
  <zorder>source_forward</zorder>
  <zorder>source_right</zorder>
  <zorder>object</zorder>
  <zorder>object_up</zorder>
  <zorder>object_down</zorder>
  <zorder>object_right_rotate</zorder>
  <zorder>object_left_rotate</zorder>
  <zorder>reciever</zorder>
  <zorder>single_shoot</zorder>
  <zorder>I</zorder>
  <zorder>time</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>U</zorder>
  <zorder>webcam_on</zorder>
  <zorder>scrollArea</zorder>
  <zorder>traject_source_left</zorder>
  <zorder>traject_source_back</zorder>
  <zorder>traject_source_forward</zorder>
  <zorder>traject_source_right</zorder>
  <zorder>traject_object_up</zorder>
  <zorder>traject_object_down</zorder>
  <zorder>traject_receiver_forward</zorder>
  <zorder>traject_receiver_back</zorder>
  <zorder>traject_receiver_left</zorder>
  <zorder>traject_receiver_right</zorder>
  <zorder>step_for_source_left</zorder>
  <zorder>step_for_object_up</zorder>
  <zorder>step_for_object_down</zorder>
  <zorder>step_for_receiver_right</zorder>
  <zorder>step_for_source_right</zorder>
  <zorder>step_for_source_back</zorder>
  <zorder>step_for_source_forward</zorder>
  <zorder>step_for_receiver_forward</zorder>
  <zorder>step_for_receiver_back</zorder>
  <zorder>step_for_receiver_left</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
